src/test/appForm.test.js

```diff
@@ -309,6 +321,55 @@ describe("App Form", function () {
           FormActions.update("instances", 2);
         });
       });
+
+      describe("the executor field", function () {
+        it("updates correctly", function (done) {
+          AppFormStore.once(FormEvents.CHANGE, function () {
+            expectAsync(function () {
+              expect(AppFormStore.fields.executor).to.equal("//cmd");
+            }, done);
+          });
+
+          FormActions.update("executor", "//cmd");
+        });
+      });
+
+      describe("the ports field", function () {
+        it("updates correctly", function (done) {
+          AppFormStore.once(FormEvents.CHANGE, function () {
+            expectAsync(function () {
+              expect(AppFormStore.fields.ports).to.equal("23, 24, 25");
+            }, done);
+          });
+
+          FormActions.update("ports", "23, 24, 25");
```

Comment:
I decided not to tests this because I tested the function of this in appFormTransforms.test.js .
The way of calling these validator functions is tested already, I don't see an advantage of having these more than twice. But we can discuss this tomorrow.